makelib/imagelight.mk

```diff
@@ -111,7 +111,7 @@ $(foreach r,$(REGISTRY_ORGS), $(foreach i,$(IMAGES),$(eval $(call repo.targets,$
 # Common Targets
 
 do.build.image.%:
-	@$(MAKE) -C $(IMAGE_DIR)/$* IMAGE_PLATFORMS=$(IMAGE_PLATFORM) IMAGE=$(BUILD_REGISTRY)/$*-$(ARCH) img.build
+	@$(MAKE) -C $(IMAGE_DIR)/$* IMAGE_PLATFORMS=$(IMAGE_PLATFORM) IMAGE=$(BUILD_REGISTRY)/$*:$(VERSION) img.build
```

Comment:
other providers and consumers of this build submodule don't seem to be having any problems with this - what's different about your provider build and how do you have confidence that changing this for everyone won't break others in the ecosystem?